crates/brix_commands/src/command/copy/tests/from.rs

```diff
@@ -0,0 +1,57 @@
+#[cfg(test)]
+mod params {
+    use std::path::PathBuf;
+
+    use dialoguer::console::Term;
+
+    use crate::command::copy::{CopyParams, OverwritableCommand};
+    use crate::{CopyCommand, ProcessedCommandParams};
```

Comment:
This line is erroring right here. It seems that reexported items in the crate root do not come into effect inside the crate...
```
use crate::{command::copy::CopyCommand, ProcessedCommandParams}
```
The above seems to work however.

Reply:
Very interesting... I removed my changes with a `git reset --hard` and this issue disappeared.
Looks like VSCode was just doing a bad.